### ISSUE TITLE
bridge=aqueduct is missing from the structure preset of waterway

### ISSUE THREAD

**Issue by SteveLz** [openstreetmap/id-tagging-schema#1464]
### Description

There are tunnels but no bridge ([aqueduct](https://wiki.openstreetmap.org/wiki/Tag:bridge%3Daqueduct)) in the structure of the waterway preset.
[taginfo](https://taginfo.openstreetmap.org/tags/bridge%3Daqueduct)

**Comment by tyrasd** [comment]
the relevant file is [`structure_waterway.json`](https://github.com/openstreetmap/id-tagging-schema/blob/main/data/fields/structure_waterway.json)

**Comment by SteveLz** [comment]
> the relevant file is [`structure_waterway.json`](https://github.com/openstreetmap/id-tagging-schema/blob/main/data/fields/structure_waterway.json)

Changed in #1273, but selecting `bridge` on a canal does not seem to automatically set `aqueduct` like selecting `tunnel` automatically sets `culvert`. The relevant code should be [here](https://github.com/openstreetmap/iD/blob/develop/modules/ui/fields/radio.js#L316), right?
By the way, is it possible to embed `bridge:name` and `tunnel:name` in structure? It may be better than separate field boxes.

**Comment by tyrasd** [comment]
yep, automatically setting `aqueduct` makes sense for waterways (similar to how `tunnel` is defaulting to `culvert`), and it needs to be implemented in iD's code.

> By the way, is it possible to embed bridge:name and tunnel:name in structure? 

Possible yes, but especially for the multilingual name field, the UI would be quite tricky to implement and work well. I'd say the approach in #1273 is good enough for the moment.
